nomos-da/kzgrs/src/common.rs

```diff
@@ -129,9 +129,9 @@ pub fn bytes_to_polynomial_unchecked<const CHUNK_SIZE: usize>(
 /// data fits within the bls modulus.
 /// Data len cannot be higher than `BYTES_PER_FIELD_ELEMENT`
 #[must_use]
-pub fn field_element_from_bytes_le(b: &[u8]) -> FieldElement {
+pub fn field_element_from_bytes_be(b: &[u8]) -> FieldElement {
```

Comment:
Should we just drop the last part, and call it `field_element_from_bytes` instead?